griptape/utils/chat.py

```diff
@@ -15,23 +16,37 @@ class Chat:
     intro_text: Optional[str] = field(default=None, kw_only=True)
     prompt_prefix: str = field(default="Q: ", kw_only=True)
     response_prefix: str = field(default="A: ", kw_only=True)
+    streaming_output_fn: Callable[[str], None] = field(
+        default=lambda x: print(x, end=""), kw_only=True
+    )
     output_fn: Callable[[str], None] = field(
         default=Factory(lambda: print), kw_only=True
     )
 
-    def start(self) -> None:
+    def start(self, initial_turn: str = None) -> None:
+        if initial_turn:
+            self.output_fn(initial_turn)
```

Comment:
What does `initial_turn` do? I think ideally we don't want to introduce another parameter.

Reply:
Thanks for persisting! In the request it was suggested that the streaming conversation would be started with

`Chat(agent, stream=True).run("Howdy!")`

I took this to mean that they'd like to be able to pass in an initial greeting to kick the conversation off.  That's what this is for.

Reply:
I just clarified with @shhlife offline, we don't need that functionality for now. Let's just focus on the streaming side of things.

Reply:
@shhlife's request still has that Chat should take a Stream parameter. Stream was removed from the utility after we decided not to mutate the structure [here (https://github.com/griptape-ai/griptape/pull/409#discussion_r1380741513)], instead we're leaving it to user to set Stream in the prompt driver and agent.

Should Chat still have Stream parameter? If so it could be used to validate that Stream is set in the prompt driver and the agent?

Reply:
Let's also stick with our original implementation. The only place the user should set `Stream` is on the Structure.